lib/stat_tracker.rb

```diff
@@ -275,4 +276,34 @@ def convert_season_id_to_game_id(season_id)
       end
       game_id_by_season.game_id
    end
+
+   def most_tackles(season)
+      tackles = Hash.new(0)
+      @data_game_teams.each do |data_game_team|
+         tackles[data_game_team.team_id] += data_game_team.tackles
+      end
+      require'pry';binding.pry
+      
+      #retruns GameTeam object with most tackles
```

Comment:
I like the commented lines that explains what the code below does, it helped me understand what's happening easier.